File: cpp_version/src/contato.cpp
#include "contato.hpp"

Contato::Contato(std::string telefone, std::string email, std::string site) {
	this->modContato(telefone, email, site);
}

void Contato::modContato(std::string telefone, std::string email, std::string site) {
	this->telefone = telefone;
	this->email = email;
	this->site = site;
}

std::string Contato::getTelefone() {
	return this->telefone;
}
std::string Contato::getEmail() {
	return this->email;
}
std::string Contato::getSite() {
	return this->site;
}

File: python/contato.py
class Contato:
    def __init__(self, telefone, email, site):
        self.telefone = telefone    #str
        self.email = email          #str
        self.site = site            #str

    #Modificar as informacoes de contato.
    def modContato(self, telefone, email, site):
        self.telefone = telefone
        self.email = email
        self.site = site

    def infoCompleta(self):
        return "\nTelefone: " + str(self.telefone) + "\nEmail: " + str(self.email) + "\nSite: " + str(self.site)
